src/theme/provider.tsx

```diff
@@ -4,13 +4,27 @@ import React, { useState, useEffect, useCallback } from "react";
 
 import { orangeTheme as defaultTheme } from "./orange";
 
+interface BodyTheme {
+  white: string,
+  gray5: string,
+  gray4: string,
+  gray3: string,
+  gray2: string,
+  gray1: string,
+  black: string;
+}
+
+export interface SharedDefaultTheme { 
```

Comment:
all themes **should** share same attributes, no sure why this is needed this

Reply:
they share all attribute, by some attributes has the same values in each theme, for example black, white and gray palette colors.